src/backend/joanie/core/serializers/admin.py

```diff
@@ -457,6 +457,13 @@ class AdminOrderGroupSerializer(serializers.ModelSerializer):
         default=models.OrderGroup._meta.get_field("is_active").default,
     )
     nb_available_seats = serializers.SerializerMethodField(read_only=True)
+    discount = serializers.SlugRelatedField(
+        slug_field="id",
+        queryset=models.Discount.objects.all(),
+        many=False,
+        required=False,
+        allow_null=True,
+    )
```

Comment:
Hmm we could assume to use a custom serializer to return the Discount object instead of its id